modul.config.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
   repoName: "bbzbl-modul-117",
   url: 'https://gaebi4102.github.io',
   organizationName: "gaebi4102", 
-}
\ No newline at end of file
+  pdfFilterKeyword: "pdf"
```

Comment:
hier kann man das keyword für die pdfs ändern. Evt, kommt in einer nächsten version dass man mehrere PDFs hier konfigurieren kann.